### PR TITLE
cleanup: correct misspelling in rbd/clone.go

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: internal/rbd/clone.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func (rv *rbdVolume) checkCloneImage(ctx context.Context, parentVol *rbdVolume)
 	err = parentVol.checkSnapExists(snap)
 	if err == nil {
 		// the temp clone exists, delete it lets reserve a new ID and
-		// create new resources for clearner approach
```
Comment (Madhu-1) [inline internal/rbd/clone.go:119]: It should be cleaner?

File: internal/rbd/clone.go [hunk 2]
```diff
@@ -116,7 +116,7 @@ func (rv *rbdVolume) checkCloneImage(ctx context.Context, parentVol *rbdVolume)
...
+		// create new resources for clearer approach
```
Comment (Yuggupta27) [inline internal/rbd/clone.go]:
```suggestion
		// create new resources for a cleaner approach
```

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]

File: internal/rbd/clone.go (hunk 1 above)
Comment (humblec) [reply internal/rbd/clone.go:119]: Yeah, I was confused too. Looks like cleaner is better.. accommodated..  Thanks  PTAL.

File: internal/rbd/clone.go (hunk 2 above)
Comment (humblec) [reply internal/rbd/clone.go]: @Yuggupta27 Thanks.. Updated. ptal

**Review by Yuggupta27 — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@humblec return-8 branch is created on the main repo can you please delete it.

**Comment by humblec** [comment]
> @humblec return-8 branch is created on the main repo can you please delete it.

@Madhu-1 not seeing it here https://github.com/ceph/ceph-csi/branches or am I missing it frm my view,

**Comment by Madhu-1** [comment]
i deleted it before few minutes

**Comment by humblec** [comment]
> i deleted it before few minutes

Ah.. Thanks :+1:

**Pull request merged**